subsys/dsp/cmsis_backend/public_overrides/zdsp_backend/dsp.h

```diff
@@ -0,0 +1,158 @@
+/* Copyright (c) 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef SUBSYS_MATH_CMSIS_BACKEND_PUBLIC_ZDSP_BACKEND_DSP_H_
+#define SUBSYS_MATH_CMSIS_BACKEND_PUBLIC_ZDSP_BACKEND_DSP_H_
```

Comment:
Do we really need `subsys/dsp/cmsis_backend/public_overrides/zdsp_backend/dsp.h`?

Can't it be something simpler like: `subsys/dsp/cmsis/dsp_backend.h`?